MeesGame/Gameplay/GameObjects/Tile.cs

```diff
@@ -223,6 +285,20 @@ public virtual InventoryItem GetItem()
         {
             return null;
         }
+
+        public abstract void UpdateGraphics();
```

Comment:
Is this function really necessary? It's currently only used in KeyTile but KeyTile could just update the graphics in the EnterTile() function